icu4j/main/classes/core/src/com/ibm/icu/impl/number/MacroProps.java

```diff
@@ -30,6 +30,7 @@
     public SignDisplay sign;
     public DecimalSeparatorDisplay decimal;
     public Scale scale;
+    public String usage; // = null;  (no usage)
```

Comment:
No default constructors, no confusion about what its default value is, I'm removing the comment.

Reply:
Done: aa5560b39a2a5c544d9208e35cb6c10326e7be4b